src/test/java/net/bhardy/bizzo/billing/PolicyBuilderTest.java

```diff
@@ -28,13 +28,23 @@
 import static net.bhardy.bizzo.billing.PolicyFilter.not;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
 
 /**
  *
  */
 public class PolicyBuilderTest {
     @Test
-    public void testBuilder() {
+    public void isDueOn_monthlySimple() {
+        BillingPolicy policy = BillingPolicy.builder()
+                .sameDayEveryMonth(5)
+                .build();
+
+        LocalDate oneSaturday = LocalDate.of(2017, 8, 5);
+        assertTrue(policy.isDueOn(oneSaturday));
+    }
+    @Test
+    public void isDueOn_monthlyFiltered() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00100)